Layers/LayerRenderersConfig.gd

```diff
@@ -5,6 +5,7 @@ var renderers = {
 	Layer.RenderType.BASIC_TERRAIN: preload("res://Layers/Renderers/Terrain/BasicTerrainRenderer.tscn"),
 	Layer.RenderType.REALISTIC_TERRAIN: preload("res://Layers/Renderers/Terrain/RealisticTerrainRenderer.tscn"),
 	Layer.RenderType.POLYGON: preload("res://Layers/Renderers/Polygon/PolygonRenderer.tscn"),
+	Layer.RenderType.GRADIENT_POLYGON: preload("res://Layers/Renderers/Polygon/GradientPolygonRenderer.tscn"),
```

Comment:
Do we need a new scene or is it possible to generically put both render-types into one? In the``GradientPolygonRenderer`` it appears that it shares a lot of code with the ``PolygonRenderer`` (which should not be named like that anyways in my opinion)

Reply:
Yeah I've been meaning to rename the PolygonRenderer to BuildingRenderer. I think it'd be good to separate them (perhaps with BuildingRenderer inheriting from PolygonRenderer) since the buildings have a lot of very specific logic.